96jonesa/AutoBought
# Change window.confirm to react-confirm-alert for deletion confirmation

This pull request should hopefully fix the Windows issue where the confirmation alert that pops up when you click the "Delete Login" or "Delete Card" buttons causes the app window to lose focus.

**To test:**
- Pull these changes and run AutoBought as normal. 
- Try deleting a payment card and a login. In both cases, a dialog box should appear in the center of the screen asking if you actually want to delete the login or payment. 
    - If you click anywhere outside the dialog, press the ESC key, or click "No", the dialog should disappear and the login or payment should remain as-is. 
    - If you click "Yes", the dialog should disappear, the login or payment should also disappear, and you should see the toast message at the top of the screen indicating that the deletion was successful.
- **Most importantly, you should be able to click and type inside the fields in any of the forms on the Settings page immediately after acknowledging this dialog!!!!**
    - If you still can't click on or type in any of the form fields, let me know and I'll figure out a different fix asap.
